package/com.unity.formats.usd/Runtime/Plugins/x86_64/plugInfo.json.meta

```diff
@@ -1,9 +0,0 @@
-fileFormatVersion: 2
-guid: 490ba57b3b3952d43bcbe916c3b56dc7
-timeCreated: 1521838944
```

Comment:
I don't see any changes to actual plugInfo.json files, which is surprising, they almost always have modifications after an upgrade.